.coderabbit.yml

```
@@ -0,0 +1,237 @@
+
+    - path: "Package.swift"
+      instructions: |
+        You are a senior Swift package maintainer reviewing Swift Package Manager configuration for KarrotCodableKit.
+
+        ## 1. Package Configuration [HIGH]
+        - Verify platform requirements match library capabilities
+        - Check dependency version constraints are appropriate
+        - Assess target dependencies and product configurations
+        - Validate macro target setup with SwiftSyntax dependencies
+
+        ## 2. Versioning and Compatibility [HIGH]  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)